middle.mkd

```diff
@@ -331,14 +331,17 @@ height in meters above or below the WGS 84 reference ellipsoid. In the
 absence of elevation values, applications sensitive to height or depth
 SHOULD interpret positions as being at local ground or sea level.
 
-Other coordinate reference systems, including ones described by CRS
-objects of the kind defined in [GJ2008] are NOT RECOMMENDED. GeoJSON
+Other coordinate reference systems are NOT RECOMMENDED. GeoJSON
 processing software SHALL NOT be expected to have access to coordinate
-reference systems databases. Applications requiring a CRS other than the
-default MUST assume all responsibility for CRS identification,
-coordinate accuracy, and interpretation of missing elevation values.
-Furthermore, GeoJSON coordinates MUST NOT under any circumstances use
-latitude, longitude order.
+reference system databases or be expected to parse and interpret the CRS
+objects described in [GJ2008]. Furthermore, to prevent coordinate order
+confusion for parsers that do not have a CRS database, GeoJSON
+coordinates MUST NOT under any circumstances use latitude, longitude
+order. For example, GeoJSON data using a coordinate reference system
```

Comment:
If the data is always lon, lat (or X, Y) then you guarantee that the data is directionally correct. It might show up on the wrong place, but at least you can be sure that you know what is north of what and what is east of what.

This has been a controversial subject for over a decade. There are communities that find this approach wholly unacceptable, but in fairness to the authors they do not represent the target audience for GeoJSON.